modules/oc-mirror-about.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="installation-oc-mirror-about_{context}"]
+= About the oc-mirror plug-in
+
+The oc-mirror OpenShift CLI (`oc`) plug-in allows you to mirror all required {product-title} content and custom images to your mirror registry by using a single tool. It provides the following features:
+
+* Provides a centralized method to mirror {product-title} releases, Operators, helm charts, and custom images.
+* Maintains update paths for {product-title} and Operators.
+* Uses a declarative image set configuration file to include only the {product-title} releases, Operators, and images that your cluster needs.
+* Performs incremental mirroring, which reduces the size of future image sets.
```

Comment:
I think that oc-mirror update operations are differential vice incremental, but that might be more of a nit.

Reply:
I had actually pulled that from the slide of the what's new presentation where Daniel talked about disconnected mirroring: https://speakerdeck.com/redhatlivestreaming/whats-new-in-openshift-4-dot-10?slide=10

Let me know if I should change it here.